CHANGELOG-1.4.md

```diff
@@ -34,23 +34,25 @@ CHANGELOG - ZIKULA 1.4.x
     - Added validation of extension composer files (#2669, #3151)
 
  - Vendor updates:
+    - assetic updated from 1.3.2 to 1.4.0
     - doctrine dbal updated from 2.5.4 to 2.5.5
     - doctrine orm updated from 2.5.4 to 2.5.5
+    - font-awesome updated from 4.6.3 to 4.7.0
     - gedmo/doctrine-extensions updates from 2.4.22 to 2.4.24
+    - jquery.mmenu updated from 5.6.3 to 5.7.6
     - jquery ui updates from 1.12.0.1 to 1.12.1
+    - justinrainbow/json-schema added in version 4.0.1
+    - jstree updated from 3.3.2 to 3.3.3
+    - knp-menu installed at version 2.2.0
+    - knp-menu-bundle installed at version 2.1.3
+    - liip/imagine-bundle installed at 1.6.0
     - psr log updated from 1.0.0 to 1.0.2
     - sensio distribution bundle updated from 5.0.8 to 5.0.14
     - sensio generator bundle updated from 3.0.7 to 3.0.11
     - sensiolabs security checker updated from 3.0.2 to 4.0.0
     - symfony updated from 2.8.9 to 2.8.13
-    - symfony service definition validator updates from 1.2.5 to 1.2.6
+    - symfony service definition validator updated from 1.2.5 to 1.2.6
     - twig updated from 1.24.1 to 1.27.0
-    - knp-menu installed at version 2.2.0
-    - knp-menu-bundle installed at version 2.1.3
-    - liip/imagine-bundle installed at 1.6.0
-    - font-awesome updated from 4.6.3 to 4.7.0
-    - added justinrainbow/json-schema 4.0.1
-    - jstree updated from 3.3.2 to 3.3.3
```

Comment:
resorted them alphabetically for easier readability (like we did also for 1.4.3)